frontend/src/components/Admin/AdminShoutouts/AdminShoutouts.tsx

```diff
@@ -120,25 +120,25 @@ const AdminShoutouts: React.FC = () => {
   };
 
   const DisplayList = (): JSX.Element => {
+    const displayedShoutouts = displayShoutouts;
+
```

Comment:
What is the purpose of assigning this state to a const?

Reply:
Oops this was probably left on accident, probably from trying to add pagination